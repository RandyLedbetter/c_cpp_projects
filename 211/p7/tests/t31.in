insert Chico Sacramento 90
insert Chico Redding 73
lookup Chico
remove Sacramento
lookup Chico
remove Redding
lookup Chico
