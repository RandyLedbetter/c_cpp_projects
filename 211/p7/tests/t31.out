Chico: none, Sacramento(90), Redding(73)
Chico: none, none, Redding(73)
Chico: none, none, none
